app/services/audit/base_replication_to_endpoint.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-module S3
+module Audit
   # Base class for AWS and IBM audit classes
-  class S3Audit
+  class BaseReplicationToEndpoint
```

Comment:
Isn't Base usually the last bit of the compound name?   I know it's long here.  If we do Audit::Replication::ToEndpointBase is that worse or better?

Reply:
In my head it is always first, but if there is a convention happy to move it.

I didn't add a `Replication` namespace to `Audit` because none of the other audits have namespaces and didn't think it helped much.